### ISSUE TITLE
Data field breaking strings and creating erroneous look up links

### ISSUE THREAD

**Issue by sebw** [ECSC/analogi#10]
This is a string as displayed in default OSSEC Web UI.

Jun 17 00:51:10 srproxy01a sshd[15100]: Did not receive identification string from 193.x.x.123

In analogi, the string is broken in the data field (while the IP field contains the correct address), and it creates a link to look up the incomplete IP address :

http://server/ossecui/detail.php?rule_id=5706&breakdown=source

1   5706    6   2013/06/17 2:55:57  (srproxy01b.x) any->/var/log/secure 193.x.x.123 Jun 17 00:55:54 srproxy01b sshd[26992]: Did not receive identification string from 193.x.x.12
1998    5706    6   2013/06/17 2:55:22  (srproxy01b.x) any->/var/log/secure 193.x.x.123 Jun 17 00:55:21 srproxy01b sshd[26964]: Did not receive identification string from 193.x.x.12

So basically, 192.168.100.123 becomes a link to look up 192.168.100.12.

Thanks for analogi, it's so cool and much better than the default UI. :+1:
